script/grblcon.py

```diff
@@ -166,6 +201,9 @@ def __init__(self):
         self._text_entry = urwid.Edit('>>> ')
         self._screen = raw_display.Screen()
 
+        # ADC poll interval
+        self.ADC_POLL_TIME = 0.2
```

Comment:
Putting a constant in the middle of the initializer like this seems odd because it will be hard to find later. I don't know where the right place to put this is, but better places include the top of the initializer, the bottom of the initializer, the top of the class (above the initializer), and near the top of the file (just below the import statements).